omim2obo/parsers/omim_txt_parser.py

```diff
@@ -233,36 +245,48 @@ def parse_mim2gene(lines, filename='mim2gene.tsv', filename2='genemap2.tsv') ->
     return gene_map, pheno_map, hgnc_map
 
 
-def parse_morbid_map(lines) -> Dict[str, List[str]]:
-    """Parse morbid map file"""
-    # todo: phenotype_mapping_key_meanings: could be useful at some point. The explanation for the keys and where to
-    #  find them within the phenotype label can be found at the bottom of morbidmap.txt
-    # phenotype_mapping_key_meanings = {
-    #     '1': 'The disorder is placed on the map based on its association with a gene, but the underlying defect is '
-    #          'not known.',
-    #     '2': 'The disorder has been placed on the map by linkage or other statistical method; no mutation has '
-    #          'been found.',
-    #     '3': 'The molecular basis for the disorder is known; a mutation has been found in the gene.',
-    #     '4': 'A contiguous gene deletion or duplication syndrome, multiple genes are deleted or duplicated causing '
-    #          'the phenotype.',
-    # }
-    ret = {}
-    p = re.compile(r".*,\s+(\d+)\s\(\d\)")
+def parse_morbid_map(lines) -> Dict[str, Dict]:
```

Comment:
This is where most of the changes happened for #73. I got an updated regex [from Kevin's code](https://github.com/monarch-initiative/monarch-ingest/blob/main/monarch_ingest/ingests/omim/gene_to_disease.py) which provided a fix for #74.